src/pages/library/Datasets.js

```diff
@@ -252,22 +251,6 @@ const hca = h(Participant, {
   }, ['Browse Data'])
 ])
 
-const nemo = h(Participant, {
```

Comment:
@katevoss did this wipe out NeMO on purpose...?

Reply:
Nope, reverting now.

Reply:
Sorry, this PR accidentally reverted a bunch of stuff back. We're rolling it back now so NeMO should be back on soon! Sorry again, and thanks!